saleor/product/migrations/0162_auto_20220209_1517.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.12 on 2022-02-09 15:17
```

Comment:
Could we merge this migration with `0159_auto_20220209_1501.py` both just adding fields?